src/database/Database.cpp

```diff
@@ -140,9 +141,23 @@ Database::applySchemaUpgrade(unsigned long vers)
             }
         }
         break;
+
     case 6:
         mSession << "ALTER TABLE peers ADD flags INT NOT NULL DEFAULT 0";
         break;
+
+    case 7:
+        Upgrades::dropAll(*this);
```

Comment:
better is to do those changes in the commits that modify the schema (only exception is SCHEMA_VERSION that is modified in the first of those commits)